Vagrantfile

```diff
@@ -90,14 +90,14 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   config.vm.provider "virtualbox" do |vb, override|
-    override.vm.box = "ubuntu/trusty64"
+    override.vm.box = "ubuntu/xenial64"
     # It's possible we can get away with just 1.5GB; more testing needed
     vb.memory = 2048
     vb.cpus = 2
   end
 
   config.vm.provider "vmware_fusion" do |vb, override|
-    override.vm.box = "puphpet/ubuntu1404-x64"
+    override.vm.box = "puphpet/ubuntu1604-x64"
```

Comment:
The "puphpet" looks like a typo. But then, no one has ever reported that it is not working with vmware.

Reply:
( ic, nvm )

Reply:
https://app.vagrantup.com/puphpet/boxes/ubuntu1604-x64